app/components/c-panel.vue

```diff
@@ -0,0 +1,419 @@
+      hasTouch() {
+        return 'ontouchstart' in document.documentElement || navigator.maxTouchPoints > 0 || navigator.msMaxTouchPoints > 0;
+      },
+    },
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+
+  .c-panel {
+    $shadow-blur-radius: 5px;
+
+    position: relative;
+    background-color: $color-grayscale--1000;
+
+    &--border-1 {
+      border-bottom: 1px solid;
+    }
+
+    &--border-2 {
+      border-bottom: 2px solid;
+    }
+
+    &--color-yellow {
+      border-color: $color-primary--1;
+    }
+
+    &--color-blue {
+      border-color: $color-secondary--1;
+    }
+
+    &--color-gray {
+      border-color: $color-grayscale--600;
+    }
+
+    /* TODO - remove when e-heading is implemented */
+    &__heading-underline {
+      @include font($font-size: $font-size--18, $line-height: 23px);
+
+      display: inline-block;
+      color: $color-secondary--1;
+      border-bottom: 1px solid $color-primary--1;
+      padding-bottom: $spacing--5;
+    }
+
+    &__heading--spacing-0,
+    &__content--spacing-0 {
+      padding: $spacing--0;
+    }
+
+    &__heading--spacing-500,
+    &__content--spacing-500 {
+      padding: $spacing--10;
+    }
+
+    &__triangle-button {
+      position: absolute;
+      bottom: 0;
+      right: 0;
+      border: none;
+      cursor: pointer;
+      overflow: hidden;
+    }
+
+    &__triangle-button--size-small {
+      width: 29px;
+      height: 29px;
+    }
+
+    &__triangle-button--size-big {
+      width: 40px;
+      height: 40px;
+    }
+
+    &__shadow {
+      position: absolute;
+      width: 100px;
+      height: 100px;
+      top: 7px;
+      right: -70px;
+      transform: rotate(138deg);
+      box-shadow: inset 0 0 $shadow-blur-radius $color-grayscale--200;
+      animation: shadowFadeOut 0.25s;
+    }
+
+    &__shadow--color-yellow {
+      background-color: $color-primary--1;
+    }
+
+    &__shadow--color-blue {
+      background-color: $color-secondary--1;
+    }
+
+    &__shadow--color-gray {
+      background-color: $color-grayscale--600;
+    }
+
+    &__shadow--hover {
+      animation: shadowFade 0.25s;
+      box-shadow: 0 0 $shadow-blur-radius $color-grayscale--200;
+    }
+
+    @keyframes shadowFade {
+      0% { box-shadow: inset 0 0 $shadow-blur-radius $color-grayscale--200; }
+      50% { box-shadow: none; }
+      100% { box-shadow: 0 0 $shadow-blur-radius $color-grayscale--200; }
+    }
+
+    @keyframes shadowFadeOut {
+      0% { box-shadow: 0 0 $shadow-blur-radius $color-grayscale--200; }
+      50% { box-shadow: none; }
+      100% { box-shadow: inset 0 0 $shadow-blur-radius $color-grayscale--200; }
+    }
+
+    &__plus {
+      display: block;
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      transform-origin: 50% 50%;
+      transform: rotate(0deg);
+      transition: 0.25s ease-out;
+    }
+
+    &__plus--size-small {
+      width: 13px;
+      height: 13px;
+    }
+
+    &__plus--size-big {
+      width: 20px;
```

Comment:
- [ ] Please use `$spacing--` variables

Reply:
I thought i leave pixel values here, because for &__plus--size-small i had to use 13px and since those cases are related... you know what i mean?